### PR TITLE
fix: wrong id type Int -> String in MongoDB one-to-one example

### PR DESCRIPTION
**Pull request by seompark**
## Describe this PR

Fix wrong id type Int -> String in MongoDB one-to-one example

## Changes

- MongoDB one-to-one relation schema example Int -> String

## What issue does this fix?

Fix typo on example.

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx [hunk 1]
```diff
@@ -261,7 +261,7 @@ model User {
 }
 
 model Profile {
-  id     Int    @id @default(auto()) @map("_id") @db.ObjectId
+  id     String    @id @default(auto()) @map("_id") @db.ObjectId
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx]:
```suggestion
  id     String  @id @default(auto()) @map("_id") @db.ObjectId
```

**Review by janpio — APPROVED** [review]

File: content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx [hunk 2]
```diff
@@ -261,7 +261,7 @@ model User {
 }
 
 model Profile {
-  id     String    @id @default(auto()) @map("_id") @db.ObjectId
+  id     String  @id @default(auto()) @map("_id") @db.ObjectId
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx]: Arghe, seems I missed one:
```suggestion
  id     String @id @default(auto()) @map("_id") @db.ObjectId
```

Comment (seompark) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx]: Thx :)

**Review by janpio — APPROVED** [review]

**Review by keerlu — APPROVED** [review]

**Comment by keerlu** [comment]
Thanks @smnis!

**Pull request merged**